data/EMSL_BASIS_SETS

```diff
@@ -32659,15 +32659,17 @@ FE NASA-Ames-cc-pVTZ
 H  Ahlrichs-def2-SVP
   2
```

Comment:
That has to be a "3" then. Otherwise, the parser will only read the first two sets.

Reply:
Thanks.